pending/bloodhound/PKGBUILD

```diff
@@ -0,0 +1,75 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+_npmname=BloodHound
+pkgname=bloodhound
+pkgver=705.124ecee
+pkgrel=1
+pkgdesc='Six Degrees of Domain Admin'
+groups=('archstrike')
+arch=('any')
```

Comment:
'any' arch doesn't work here since it needs to be built differently for different arches. you need to explicitly write them